content/general/versions/v1.0/push.textile

```diff
@@ -19,7 +19,7 @@ Ably's push notification service is currently in beta. Please bear in mind:
 
 * Whilst the API we provide now is planned for our GA release, *the API is subject to change*. However, we will endeavour to only make changes to the API where necessary before we transition the service to GA (general availability).
 * The push service, as far as we are aware, is ready for production levels of usage. However, given the beta status, push notifications have not yet been stress tested under real-world production loads. As a result, you should expect some bugs. If you experience any problems, please "get in touch":https://ably.com/contact and we'll work as quickly as possible to help identify the problem and fix it.
-* The client libraries currently have varying levels of support for push notifications. Please refer to the "push notification client library download section":/general/push/download for details on how to use push on your devices and servers now.
+* The client libraries currently have varying levels of support for push notifications.
```

Comment:
I'm just wondering why we have removed this link entirely? `/general/push/#download` seems to link to this info?

Reply:
`/general/push/#download` only exists for version 1.2 so I thought it better to remove the link entirely than link to a different version (without someone necessarily being aware the version changed). Happy to restore it though if you disagree!

Reply:
Understood!